selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/theia/TheiaIde.java

```diff
@@ -70,6 +81,48 @@
   @FindBy(xpath = Locators.ABOUT_DIALOG_OK_BUTTON_XPATH)
   WebElement aboutDialogOkButton;
 
+  private String getNotificationMessageEqualsToXpath(String messageText) {
```

Comment:
`getNotificationMessageEqualsToXpath(String messageText)` could be simplified to `getNotificationEqualsToXpath(String notification)`